### PR TITLE
Use /var/opt as default for storage

### PR DESCRIPTION
**Pull request by otoolep**

### CODE REVIEW

File: etc/config.sample.toml [hunk 1]
```diff
@@ -63,13 +63,13 @@ enabled = false
 # consensus.
 [broker]
 # Where the Raft logs are stored. The user running InfluxDB will need read/write access.
-dir  = "/tmp/influxdb/development/raft"
+dir  = "/var/opt/influxdb/raft"
```
Comment (jwilder) [inline etc/config.sample.toml]: PR title says `/var/lib`?  `/var/opt` is intentional?

**Comment by otoolep** [comment]
Fixed title.

**Comment by dgnorton** [comment]
lgtm  :shipit:

**Comment by jwilder** [comment]
:+1:

**Comment by otoolep** [comment]
Thanks @jwilder @dgnorton -- merging on green.

**Pull request merged**